person.rb

```diff
@@ -1,17 +1,18 @@
 require './nameable'
 
 class Person < Nameable
+  attr_reader :id, :rentals
```

Comment:
- [ ] Just like I mentioned above let's set the `rentals` as an attribute accessor. Because will might want to set the rentals and read it as well. 

- [ ] Secondly, I will suggest, you add a method for adding a rental to this class.

See example:
```ruby
def add_rental(book, date)
    # *The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association
    Rental.new(date, book, self)
end
```

Please note that the `self` keyword is referring to the `person` class

Reply:
✅RESOLVED: 
- `rentals` has been set as `attr_accessor`
- `add_rental` method has been added to the `Person` class.